addons/sms/models/ir_config_parameter.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    def _neutralize(self):
+        super()._neutralize()
+        self._neutralize_iap_service('sms.endpoint')
```

Comment:
I'm not sure the helper is necessarily that helpful as-is: `sms` has its own endpoint, does the generic IAP test endpoint actually work for it?